[2022-04-14 02:25:26,898] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: LocalIngestionDAG.ingest scheduled__2022-04-02T06:00:35+00:00 [queued]>
[2022-04-14 02:25:26,909] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: LocalIngestionDAG.ingest scheduled__2022-04-02T06:00:35+00:00 [queued]>
[2022-04-14 02:25:26,910] {taskinstance.py:1238} INFO - 
--------------------------------------------------------------------------------
[2022-04-14 02:25:26,910] {taskinstance.py:1239} INFO - Starting attempt 9 of 9
[2022-04-14 02:25:26,911] {taskinstance.py:1240} INFO - 
--------------------------------------------------------------------------------
[2022-04-14 02:25:26,927] {taskinstance.py:1259} INFO - Executing <Task(PythonOperator): ingest> on 2022-04-02 06:00:35+00:00
[2022-04-14 02:25:26,932] {standard_task_runner.py:52} INFO - Started process 7001 to run task
[2022-04-14 02:25:26,936] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'LocalIngestionDAG', 'ingest', 'scheduled__2022-04-02T06:00:35+00:00', '--job-id', '2042', '--raw', '--subdir', 'DAGS_FOLDER/data_ingestion_local.py', '--cfg-path', '/tmp/tmpzdb3rxwv', '--error-file', '/tmp/tmpjrvr9etf']
[2022-04-14 02:25:26,937] {standard_task_runner.py:77} INFO - Job 2042: Subtask ingest
[2022-04-14 02:25:27,013] {logging_mixin.py:109} INFO - Running <TaskInstance: LocalIngestionDAG.ingest scheduled__2022-04-02T06:00:35+00:00 [running]> on host f13cd06666dc
[2022-04-14 02:25:27,071] {logging_mixin.py:109} WARNING - /home/***/.local/lib/python3.7/site-packages/***/utils/context.py:152 AirflowContextDeprecationWarning: Accessing 'execution_date' from the template is deprecated and will be removed in a future version. Please use 'data_interval_start' or 'logical_date' instead.
[2022-04-14 02:25:27,098] {taskinstance.py:1426} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=LocalIngestionDAG
AIRFLOW_CTX_TASK_ID=ingest
AIRFLOW_CTX_EXECUTION_DATE=2022-04-02T06:00:35+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-04-02T06:00:35+00:00
[2022-04-14 02:25:27,099] {logging_mixin.py:109} INFO - yellow_taxi_2022_04 /opt/***/output_2022-04.csv 2022-04-02T06:00:35+00:00
[2022-04-14 02:25:27,204] {logging_mixin.py:109} INFO - connection established successfully, inserting data...
[2022-04-14 02:25:27,675] {logging_mixin.py:109} INFO - inserted the first chunk, took 0.471 second
[2022-04-14 02:25:27,676] {logging_mixin.py:109} INFO - completed
[2022-04-14 02:25:27,677] {python.py:175} INFO - Done. Returned value was: None
[2022-04-14 02:25:27,691] {taskinstance.py:1277} INFO - Marking task as SUCCESS. dag_id=LocalIngestionDAG, task_id=ingest, execution_date=20220402T060035, start_date=20220414T022526, end_date=20220414T022527
[2022-04-14 02:25:27,752] {local_task_job.py:154} INFO - Task exited with return code 0
[2022-04-14 02:25:27,787] {local_task_job.py:264} INFO - 0 downstream tasks scheduled from follow-on schedule check
